### PR TITLE
updated the file with the new requirements

### PR DESCRIPTION
**Pull request by iTrip23**
## What does this change?

Module:
Week(s):

## Description

I have updated requirements from google classroom to this file.

### CODE REVIEW

File: docs/git/homework.md [hunk 1]
```diff
@@ -78,13 +78,12 @@ Again, go to this repository and go to the "Pull Requests" tab.
...
 https://github.com/CodeYourFuture/GitHomeworkFixErrors
 
-Here you will find all of your Classmates answers to the homework. 
+Here you will find all of your Classmates answers to the homework.
 
-1. Choose two pull requests
-2. Make sure they don't have any comment on them already from other students
-3. Leave comments on any of the questions you think they have got wrong. Not sure if their answer is right? Ask them to check it and confirm.
-4. Leave a comment saying you have finished you're review
+1. Choose two pull requests with no, or few, pull requests
```
Comment (illicitonion) [inline docs/git/homework.md]:
```suggestion
1. Choose two pull requests with no, or few, comments.
```

File: docs/git/homework.md [hunk 2]
```diff
@@ -78,13 +78,12 @@ Again, go to this repository and go to the "Pull Requests" tab.
...
+2. Leave comments on any of the questions you think they have got wrong. Not sure if their answer is right? Ask them to check it and confirm.
+3. Leave a comment saying you have finished you're review
```
Comment (illicitonion) [inline docs/git/homework.md]:
```suggestion
3. Leave a comment saying you have finished your review.
```

**Review by illicitonion — APPROVED** [review]
Thanks for putting this together! It looks good - I made a couple of really minor suggestions, and I'll merge after you accept and apply them :)

File: docs/git/homework.md (hunk 1 above)
Comment (frosas) [reply docs/git/homework.md]: Lol, that was me adding a new mistake while fixing another one 🤦‍♂️

**Pull request merged**